keyboards/ploopyco/madromys/config.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2023 Colin Lam (Ploopy Corporation)
+ * Copyright 2020 Christopher Courtney, aka Drashna Jael're  (@drashna) <drashna@live.com>
+ * Copyright 2019 Sunjun Kim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// #define ROTATIONAL_TRANSFORM_ANGLE 0
+#define POINTING_DEVICE_INVERT_Y
+
```

Comment:
Gipetto's suggestion from the ploopy discord is quite valid and does improve trackball usage:

`#define PMW33XX_LIFTOFF_DISTANCE 0x00`

But i'd be interested to hear @daskygit and @drashna 's thoughts on this.

Reply:
Looking into datasheets this is a can of worms and potentially messy to include in a stock firmware. Will rescind this for now.

Reply:
Honestly, the whole liftoff distance thing is messy.  It supports more than what the datasheet indicates. 

Eg, I need to use 0xF (0b1111) on my charybdis to get proper tracking, which is outside of the 2 bits that the datasheet reports.  Between this, and the angle tune, and some other register weirdness, I'm inclined to believe that the datasheet is either out of date, or (most likely) poorly written. 

So if `##define PMW33XX_LIFTOFF_DISTANCE 0x00` seems to work better, then I think it should be fine.

Reply:
I'm doing testing right now. The best results appear to be from `#define PMW33XX_LIFTOFF_DISTANCE 0x03`, which has defined behaviour and gives the maximum liftoff height. I'll probably update this PR once I have definitive results tomorrow.

Reply:
Fixed in [cb93979](https://github.com/qmk/qmk_firmware/pull/21989/commits/cb93979b8cc002c13a68ea800ad72e9e94b44458).

For the record, I thoroughly tested values of 0x00, 0x02, 0x03, and 0xF. The best performance by far was 0x00, and this was tested across a number of PCBs, a number of case designs and ball heights, and a number of different plastics.